auth-web/src/components/auth/TeamForm.vue

```diff
@@ -37,67 +44,63 @@ import { mapActions, mapState } from 'vuex'
 import OrgModule from '@/store/modules/org'
 import { getModule } from 'vuex-module-decorators'
 
-@Component({
-  computed: {
-    ...mapState('org', ['organizations'])
-  },
-  methods: {
-    ...mapActions('org', ['createOrg']),
-    ...mapActions('org', ['syncOrganizations'])
-  }
-})
+  @Component({
```

Comment:
I see that you adjusted the spacing for the rest of the file to be 2 (which is correct).  I think this component block should remain at it's current indentation level though.

Reply:
is it better now? I think my IDE continuation intend was wrong...

Reply:
Looks good now